docling_sdg/qa/critique.py

```diff
@@ -31,24 +31,9 @@ def __init__(
         self,
         critique_options: CritiqueOptions,
     ):
-        self.options = critique_options
-
-        temp: float = 0.0
-        if self.options.additional_params:
-            temp = TypeAdapter(float).validate_python(
-                self.options.additional_params.get(GenTextParamsMetaNames.TEMPERATURE)
-            )
```

Comment:
@ceberam I had to remove this for the `OpenAiLike` class - it gave me an error message saying that custom validators aren't supported.